script.js

```diff
@@ -1,8 +1,8 @@
-const slideMenu = document.querySelector('.slide_menu');
+const slideMenu = document.querySelector('.slide-menu');
 const slide = document.querySelectorAll('.slide')[1];
 const container = document.querySelector('.container');
 const btnStart = document.querySelector('.start-btn');
-const rectangleAll = document.querySelectorAll('.rectangle');
+const rectangleAll = document.querySelectorAll('.list-level__rectangle');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/game-bag/pullRequest?prid=7696171)
```suggestion
const rectangleAll = document.querySelectorAll(".list-level__rectangle");
```